lab2.js

```diff
@@ -50,8 +50,40 @@ function assert(expression, failureMessage) {
  TODO: Then, use a loop to calculate how long it took the blob to finish
  with Dowington. */
 
+function Blob(name) {
+  this.newBlob = name;
+  this.hoursToOoze = function(population, peoplePerHour) {
```

Comment:
this should be on the prototype. So, instead of defining it here, you can just make your Blob constructor, and then later assign the hoursToOoze function to the Blob prototype. You have it below; you just commented it out.